### ISSUE TITLE
[Super Epic] [CAIA A11y Support] - Benefits and Claims Maintenance Audit

### ISSUE THREAD

**Issue by coforma-terry** [department-of-veterans-affairs/va.gov-team#61722]
## About your team

- Team name: **Benefits and Claims**
-	OCTO-DE product owner: Matt Self
-	Product manager: Laura Steel
-	Designer: Fiorella Geedi
-	FE engineer: Thomas Blackwell
- Accessibility specialist:
- Content writer: 
-	Product/team Slack channel:
 
### What does your team need support for? Check all that apply.
- [ ] Launching one or more unauthenticated pages, online forms, apps, or tools on VA.gov
- [ ] Making revisions to one or more existing VA.gov pages
- [ ] Accessibility review of wireframes and/or prototypes
- [x] Accessibility issue (general request)
- [ ] Accessibility PDF review request
- [ ] Accessibility user research support ([submit a separate, additional request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=coforma-terry%2Cangelafowler82&labels=508%2FAccessibility%2C+sitewide+CAIA%2C+sitewide+accessibility%2C+CAIA-a11y-research&template=caia-a11y-research.yaml&title=%5BCAIA+A11y+Research%5D%3A+Team+name%2C+product+name))
- [ ] DST component or pattern 
- [ ] Something else 
 
### Updated Ticket Below with CAIA Team Inventory
- 4 PDFs (Non-CAIA)
    - 21-686c
    - 21-674
    - 21P-509
    - 28-1900 (CH31/VRE)
- **4 Digital Forms**
- [x] 21-686c
- [x] 21-674 
- [x] 28-1900 (CH31/VRE)
- [x] 28-8832 (Ch36/PCPG)
- **21 Issues**
    - _These existing defects (open tickets)will need addressed first, then we can proceed with an audit._ 
- [x] #16988 
- [ ] #16784 -- reviewed and error still persists
- [x] #16745 
- [x] #16711 
- [x] #22881
- [ ] #22880 
- [x] #22876 
- [x] #22874 
- [x] #22846 
- [x] #22840 
- [x] #22831 
- [ ] #22826 
- [ ] #22824
- [ ] #22653
- [x] #7759 
- [ ] #7403 
- [x] #21464 
- [x] #30026 
- [x] #39225 
- [x] #41108 

#### [CAIA A11y Support]: Benefits and Claims, Dependents #61685
- **Maintenance audit for forms:**
    - 21-686c - PDF & Digital
    - 21-674 - PDF & Digital
    - 21P-509 - PDF only
- 3 PDFs (non-CAIA)
- 2 Digital Versions
- 5 open issues (needing closed first); 2 closed 

#### [CAIA A11y Support]: Benefits and Claims, Pension #61716 - **CLOSED**

- **Maintenance audit for the following has been **canceled**:**
    - 21P-534EZ (application for dependent and indemnity compensation)- **Paper Form Only**
    - 0 PDFs
    - 0 Digital Versions
- 2 open issues (needing closed first); 12 closed

#### [CAIA A11y Support]: Benefits and Claims, CH 31/VRE #61718
- **Maintenance audit of:**
    - 28-1900 (CH31/VRE)     
- 1 PDF (non-CAIA)
- 1 Digital Version
- 10 open issues (needing closed first); 12 closed

#### [CAIA A11y Support]: Benefits and Claims, CH36/PCPG #61719
- **Maintenance audit of:**
    - 28-8832 (Ch36/PCPG) - DIGITAL ONLY
- 0 PDFs (non-CAIA)
- 1 Digital Version
- 4 open issues (needing closed first); 10 closed issues  

## Timeframe
July 31

 ### Will this new product be released incrementally (for instance 25% of users initially)? 
  - [ ] Yes
  - [ ] No
 
**Note:** If you checked yes, we'll reach out to discuss details about the content in the react widget.
  
### When do you expect to launch your product to 100% of users?
  
*Please provide an estimated date so our team can create other relevant tickets.*

### When does this work need to be completed?
- Estimated launch date:
- Estimated staging review date:
- Content, Accessibility, and IA work needed by: 

### Will this work be going through the Collaboration Cycle?
- [ ] Yes 
- [ ] No

### If no ...
- Skip to the _supporting artifacts_ section, below.
 
#### Please provide the link to your Collaboration Cycle ticket: 
- Ticket (number): 
- Collab. Cycle Ticket Opened (date): 
 
#### Please check all of the phases in the Collaboration Cycle you have completed: 
- [ ] Design Intent
- [ ] Midpoint Review
- [ ] Staging
 
##### Design Intent
- [x] Accessibility
- [ ] Content
- [ ] IA
- [ ] Translation
- [ ] Already completed, no assistance needed.
- [ ] Unknown at this time.
##### Midpoint
- [ ] Accessibility
- [ ] Content
- [ ] IA
- [ ] Translation
- [ ] Already completed, no assistance needed.
- [ ] Unknown at this time.
##### Staging
- [ ] Accessibility
- [ ] Content
- [ ] IA
- [ ] Translation
- [ ] Already completed, no assistance needed.
- [ ] Unknown at this time.
  
## Supporting artifacts
*Please provide supporting artifacts as available.*
[Product outline for 527](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/pension/README.md) (There is no outline for 534EZ
[User flows](https://app.mural.co/t/departmentofveteransaffairs9999/m/departmentofveteransaffairs9999/1683826730324/61b509f935adaa2acf2f7769a5ba911726a7b237?sender=u0e8ac1d6d7681bb7e1b80558)
[Prototype or mockups](Insert URL here between the parenthesis)
[Epics/issue that may be helpful](https://github.com/department-of-veterans-affairs/va.gov-team/issues?q=label%3Apension+)
[Form 21P-534EZ (application for dependent and indemnity compensation)](https://www.va.gov/find-forms/about-form-21p-534ez/)

- Link to product outline:
- Link to or attach user flows:
- Link to prototype or mockups:
- Link to your research plan: 
- Link to any epics/issues that may be helpful:

#### Do you plan to bring this to an upcoming content office hours session?
- [ ] Yes, benefit content office hours (Thursdays, 3:00 p.m. to 3:45 p.m. ET)
- [ ] Yes, health  content office hours (Thursdays, 11:00 a.m. to 11:30 a.m. ET)
- [ ] Yes, unauth office hours (Mondays, 11:00 a.m. to 11:30 a.m. ET)
- [ ] No, but I'd like to schedule time to talk about this request
- [ ] No, let's work asynchronously and meet if needed

 **Note:** If we think this work would benefit from a collaborative session with you, we may ask you to bring it to office hours or set up a separate time to meet.
 
### Accessibility Help in Slack
The [#accessibility-help channel](https://dsva.slack.com/archives/C8E985R32) in the DSVA Slack, is also available to you as a resource. Everyone is welcome to ask questions or get help from our a11y (accessibility) specialists. Tag `@Terry Nichols` to get a11y help asap.

## Next steps

**Once you’ve submitted this ticket, please post a link to this issue in the [#sitewide-content-ia](https://dsva.slack.com/channels/sitewide-content-ia) Slack channel and tag `@Terry Nichols`.**

- If you also need engineering support from the Public Websites team, fill out their [intake request form](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=Public+Websites%2C+Sitewide+content&labels=vsa-public-websites%2C+vsa%2C+vsa-public-websites-intake%2C+sitewide-content%2C+needs-grooming&template=public-websites-intake.md&title=%3CType+of+Request%3E+from+%3CTeam%3E).
  
- If you need a page/URL redirected, a URL changed or a vanity URL set up, please submit a [Redirect, URL change, or vanity URL request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=mnorthuis&labels=ia&template=redirect-request.md&title=Redirect+Request)
 
 - If you are creating an [experimental design,](https://design.va.gov/about/contributing-to-the-design-system/experimental-components-and-patterns#what-is-an-experimental-component-or-pattern?) please [also contact the DST. ](https://design.va.gov/about/contributing-to-the-design-system/suggest-an-addition-or-update)
- If you are requesting [a11y support for user research,](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=coforma-terry%2Cangelafowler82&labels=508%2FAccessibility%2C+sitewide+CAIA%2C+sitewide+accessibility%2C+CAIA-a11y-research&template=caia-a11y-research.yaml&title=%5BCAIA+A11y+Research%5D%3A+Team+name%2C+product+name) please complete this form, next.

**Comment by sara-amanda** [comment]
@coforma-terry updated the body of the ticket to identify the number of forms and quantity of outstanding issues relating to the forms, along with assignments so we can work with Kate on that.

**Comment by sara-amanda** [comment]
@coforma-terry  updated EPIC to reflect conversation with Martha today 7/12/2023:
- Breaking out the PDFs (4) and Digitized Forms (4).
    -  Work needing to be done includes digitized forms only. 
    - The PDFs need sent to the VBA (Veterans Benefits Administration). 
    - A PDF request can also be put into the 508 office.  
- Audit can begin, once the 21 open issues found on the last edit have been reviewed and closed out by the Benefits & Claims Team.

**Comment by tblackwe** [comment]
@sara-amanda I discussed these issues with @joshkimux and the recommendation was to request a new audit due to the age of the tickets. Theses issues originally assigned to a different team, some of them originating in 2020. 
We can start working on them, but my concerns are that: 
- the previous team did the work and didnt close the ticket, leading to confusion on our side
- Accessibility guidelines have changed since tickets were opened and we will implement the wrong solution, or address them in incorrect order due to changed severity

Please confirm that we need to resolve these issues before an audit will occur.

**Comment by sara-amanda** [comment]
@tblackwe @steele-lm - understood. Thanks for the additional info.! Our new CAIA team member, @ohgreatkate has jumped in with her benefits background, and is working on closing those, to clear them out, I believe.

**Comment by coforma-jamie** [comment]
Hi @tblackwe and @steele-lm! I've been working on [this audit](#61685), but I see here that there are still a bunch of a11y defect tickets that need to be closed (see "21 issues" at the top of this ticket). I can't complete my audit until the open tickets are closed, or their statuses are reviewed. Can you take a look at those?

**Comment by steele-lm** [comment]
@coforma-jamie Thanks for reaching out. Thomas and I outlined the intent of these audits earlier in the comments. Many of the outstanding tickets are over a year old, so the goal with the new audit request was to have CAIA review the old outstanding issues to ensure they are still issues and that the accessibility guidelines outlined in the original tickets are still valid.

**Comment by coforma-jamie** [comment]
@steele-lm CAIA would be happy to review once these tickets have been resolved or otherwise closed. We can run an audit on new work, but cannot validate existing open tickets. Since our team is so new, we understand that there may be confusion around what we do - we’re working on updating our documentation about this.

cc @sara-amanda

**Comment by steele-lm** [comment]
@coforma-jamie Thanks! For issues that are over a year old, is there any concern that we may be implementing depreciated standards that we'll need to revisit after a new audit is completed? I don't have a good sense of how often standards change.

**Comment by coforma-terry** [comment]
Tagging @sara-amanda ?

**Comment by artsymartha68** [comment]
> @coforma-jamie Thanks! For issues that are over a year old, is there any concern that we may be implementing depreciated standards that we'll need to revisit after a new audit is completed? I don't have a good sense of how often standards change.

Jumping in to say that this shouldn't be an issue! OCTO complies with WCAG 2.1 which has been in place many years (2.2 is published but we haven't "upgraded" yet). As long as the product code hasn't changed in the intervening time, the issues should still be valid.

**Comment by ohgreatkate** [comment]
Separate tickets were created for each form listed above, and feedback has been shared within each form's separate ticket.
[CAIA A11y Support]: Benefits and Claims, Dependents https://github.com/department-of-veterans-affairs/va.gov-team/issues/61685
Maintenance audit for forms:
21-686c & 21-674

[CAIA A11y Support]: Benefits and Claims, CH 31/VRE https://github.com/department-of-veterans-affairs/va.gov-team/issues/61718
Maintenance audit of:
28-1900 (CH31/VRE)

[CAIA A11y Support]: Benefits and Claims, CH36/PCPG https://github.com/department-of-veterans-affairs/va.gov-team/issues/61719
Maintenance audit of:
28-8832 (Ch36/PCPG)
